### PR TITLE
[thread-cert] refactor case 6.5.1 and 6.5.3 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

**Comment by jwhui** [comment]
@FiveDimensions , please rebase.

File: tests/scripts/thread-cert/Cert_6_5_01_ChildResetReattach.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2016, The OpenThread Authors.
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_6_5_01_ChildResetReattach.py:3]:
```suggestion
#  Copyright (c) 2020, The OpenThread Authors.
```

File: tests/scripts/thread-cert/Makefile.am [hunk 2]
```diff
@@ -94,8 +94,9 @@ EXTRA_DIST                                                         = \
     Cert_6_3_02_NetworkDataUpdate.py                                 \
     Cert_6_4_01_LinkLocal.py                                         \
     Cert_6_4_02_RealmLocal.py                                        \
-    Cert_6_5_01_ChildResetSynchronize.py                             \
+    Cert_6_5_01_ChildResetReattach.py                             \
```
Comment (jwhui) [inline tests/scripts/thread-cert/Makefile.am:97]: Align `\`.

File: tests/scripts/thread-cert/Makefile.am [hunk 3]
```diff
@@ -94,8 +94,9 @@ EXTRA_DIST                                                         = \
...
     Cert_6_5_02_ChildResetReattach.py                                \
+    Cert_6_5_03_ChildResetSynchronize.py                                \
```
Comment (jwhui) [inline tests/scripts/thread-cert/Makefile.am:99]: Align `\`.

File: tests/scripts/thread-cert/Makefile.am [hunk 4]
```diff
@@ -268,8 +269,9 @@ check_SCRIPTS                                                      = \
     Cert_6_3_02_NetworkDataUpdate.py                                 \
     Cert_6_4_01_LinkLocal.py                                         \
     Cert_6_4_02_RealmLocal.py                                        \
-    Cert_6_5_01_ChildResetSynchronize.py                             \
+    Cert_6_5_01_ChildResetReattach.py                             \
```
Comment (jwhui) [inline tests/scripts/thread-cert/Makefile.am:272]: Align `\`.

File: tests/scripts/thread-cert/Makefile.am [hunk 5]
```diff
@@ -268,8 +269,9 @@ check_SCRIPTS                                                      = \
...
     Cert_6_5_02_ChildResetReattach.py                                \
+    Cert_6_5_03_ChildResetSynchronize.py                                \
```
Comment (jwhui) [inline tests/scripts/thread-cert/Makefile.am:274]: Align `\`.

**Pull request merged**